src/js/components/login-controls.js

```diff
@@ -66,9 +66,9 @@ class LoginButton extends Component {
 	  return (
 		<div className="login-box">
 		  <form onSubmit={this.handleSubmit}>
-			  <input type="text" className="login-input" data-test="username" id="user-name" placeholder="Username" value={this.state.username} onChange={this.handleUserChange} />			  
-			  <input type="password" className="login-input" data-test="password" id="password" placeholder="Password" value={this.state.password} onChange={this.handlePassChange} />			  
-			  <input type="submit" className="login-button" value="LOGIN" />
+			  <input type="text" className="form_input" data-test="username" id="user-name" placeholder="Username" value={this.state.username} onChange={this.handleUserChange} />			  
+			  <input type="text" className="form_input" data-test="password" id="password" placeholder="Password" value={this.state.password} onChange={this.handlePassChange} />			  
```

Comment:
Why the change from `password` type to `text` on a password form input? 

Also, why the className change? That seems like it _could_ break tests for anyone not going by the better ID. I don't have a strong opinion here just trying to understand.

Reply:
I thought this might come up. I should have mentioned it. I'm not concerned about the password being stolen or observed so I don't think we need that little layer of obfuscation. I also found that as I was typing the password in, I didn't know if I had entered it in correctly. I err'd on the side of the user not being hindered at all during the login process. If it's a problem let me know. 

The form_input class is a reusable, and standard class name for objects like form inputs. Adding a class for individual elements isn't the best practice unless they are drastically different. Helps keep us from contracting "Class-itis." I tried to standardize all of the buttons in the app. For instance "remove-from-cart" button is actually just the secondary button style, and can be referenced by .btn-secondary along with the back button in the cart. In other words, individual elements should try to take advantage of more global styles, and specifically targeting individual elements that are reusable (buttons, form inputs) might not be the best approach. I would also recommend using ID's vs classes for tests if possible.

Reply:
I think the only problem with not having input type `password` is that we'd be deliberately doing something a user should _almost without exception_ never do. 

I do think that the current username/password combinations are annoying on keyboard and especially mobile keyboard because `_` is far less accessible than `-`; and the separator is used in both username and password. I'd think changing the password to address the typing password problem might be more desirable? (of course that'd highly likely break downstream CI - but better sooner than later)

Reply:
Sounds like you'd prefer to leave it as password. Not a problem at all! I'll swap this back.